Lets Do This/Networking/DSOAPI.h

```diff
@@ -32,12 +32,12 @@
 
 - (void)postUserAvatarWithUserId:(NSString *)userID avatarImage:(UIImage *)avatarImage completionHandler:(void(^)(id))completionHandler errorHandler:(void(^)(NSError *))errorHandler;
 
-- (void)createCampaignSignupForCampaign:(DSOCampaign *)campaign completionHandler:(void(^)(DSOCampaignSignup *))completionHandler errorHandler:(void(^)(NSError *))errorHandler;
-
 - (void)postReportbackItem:(DSOReportbackItem *)reportbackItem completionHandler:(void(^)(NSDictionary *))completionHandler errorHandler:(void(^)(NSError *))errorHandler;
 
 - (void)loadUserWithUserId:(NSString *)userID completionHandler:(void(^)(DSOUser *))completionHandler errorHandler:(void(^)(NSError *))errorHandler;
 
+- (void)postSignupForCampaign:(DSOCampaign *)campaign completionHandler:(void(^)(DSOCampaignSignup *))completionHandler errorHandler:(void(^)(NSError *))errorHandler;
```

Comment:
We'll want to remove this method entirely if we stick with posting signups in React Native instead of `AFNetworking` -- but it'd be cleaner to get this working instead (contains Google Analytics code, also ideal to show `SVProgressHUD` to avoid any confusion if the post errors out (user may assume the signup post worked). 

Also brings to mind how it'd be nice to get that Message Broker API up and running to treat Signups as events...